src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -483,26 +483,27 @@ void OrbitMainWindow::SetupHintFrame() {
 }
 
 void OrbitMainWindow::SetupTargetLabel() {
-  auto* target_widget = new QWidget();
-  target_widget->setStyleSheet(QString("background-color: %1").arg(kMediumGrayColor));
+  target_widget_ = new QWidget(this);
+  target_widget_->setAccessibleName("Target");
+  target_widget_->setStyleSheet(QString("background-color: %1").arg(kMediumGrayColor));
   target_label_ = new TargetLabel{};
   target_label_->setContentsMargins(6, 0, 0, 0);
   auto* disconnect_target_button = new QPushButton("End Session");
   auto* target_layout = new QHBoxLayout();
   target_layout->addWidget(target_label_);
   target_layout->addWidget(disconnect_target_button);
   target_layout->setMargin(0);
-  target_widget->setLayout(target_layout);
+  target_widget_->setLayout(target_layout);
 
-  ui->menuBar->setCornerWidget(target_widget, Qt::TopRightCorner);
+  UpdateTargetLabelPosition();
```

Comment:
I assumed that setting the widget as a corner widget guaranteed the same height as the menu bar. Was that the case? If yes, is that still guaranteed?

Reply:
Hm that is a good question. It looks like both were the same height by coincidence, but to be safe, I've added a line to explicitly set the target widget height to be equal to the menu height.